ATACseq_BonusTutorial.ipynb

```diff
@@ -456,9 +455,9 @@
     "<details>\n",
     "  <summary>Click for help</summary>\n",
     "    \n",
-    "  ```  \n",
-    "  bowtie2 -p $numthreadsint -x Tutorial1/RefGenome/GRCz11 -1 Tutorial1/Trimmed/Liver_trimmed_R1.fastq.gz -2 \n",
-    "  ```\n",
+    "\n",
+    "  `bowtie2 -p $numthreadsint -x Tutorial1/RefGenome/GRCz11 -1 Tutorial1/Trimmed/Liver_trimmed_R1.fastq.gz -2 `\n",
```

Comment:
missing file path for -2 parameter and output, update command to look like so: **`bowtie2 -p $numthreadsint -x Tutorial1/RefGenome/GRCz11 -1 Tutorial1/Trimmed/Liver_trimmed_R1.fastq.gz -2 Tutorial1/Trimmed/Liver_trimmed_R2.fastq.gz -S Tutorial1/Mapped/Liver.sam`**

Reply:
This one should be fixed in the latest commit. I'm not sure how that second argument got truncated like that. It definitely was there when I had it as a code block and something weird happened when I converted it to markdown. Good catch.